### PR TITLE
version and release note updates

### PR DESCRIPTION
**Pull request by mattkasun**
for release v0.20.1

### CODE REVIEW

File: release.md [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
+- 
     
 ## whats fixed
-- Caddy does not handle netmaker exporter well for EE
+- enrollment keys for non-admins 
+- client version displayed correctly in UI
+- upd hole punching improvments
+- SSL fallback to letsencrypt
+- permission handling for non-admin users
+
```
Comment (Aceix) [inline release.md:13]: Additional fixes:
- Client QR code and downloadable config file name
- Improved stability in Netmaker UI (null host/node errors)

Comment (mattkasun) [reply release.md:13]: Those items are more applicable to netmaker-ui release notes are they not?

Comment (Aceix) [reply release.md:13]: Yes, UI. Will we start having release notes for the UI?

**Review by Aceix — APPROVED** [review]

File: release.md (hunk 1 above)
Comment (mattkasun) [reply release.md:13]: manual for now as there is no ui release workflow yet

**Review by afeiszli — APPROVED** [review]

**Review by TobiaszCudnik — APPROVED** [review]

**Pull request merged**